verification/curator-service/ui/src/components/LinelistRow.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Case } from "../../../api/src/controllers/cases";
+
+interface RowProps {
+    case: Case
+}
+
+export default class LinelistRow extends React.Component<RowProps, {}> {
```

Comment:
Nit, I'd name this LinelistCaseRow in case we ever want to show something else in the table (separator, summary rows etc).
(This was a pattern I saw in the official react toturial I did this weekend)

(yes I'm ignoring your review guidelines to only look at /api don't blame me I'm curious :) )

Reply:
SG, I considered the same myself.

Feel free to review anything! I appreciate it; just a suggestion because I felt bad re: large PR.